doc/GuideForCommitters.md

```diff
@@ -88,9 +88,24 @@ If the change affects multiple platforms or 64-bit Windows, you must initiate a
 request build on all affected platforms prior to merging.  To launch a pull request
 build, add a comment to the pull request that follows the syntax:
    ```
-   @genie-omr build { all | zlinux | zos | plinux | aix | xlinux | win }
+   @genie-omr build [ all | {platform-list} ]
    ```
-   Combinations of platforms can be specified separated by commas.
+Where `{platform-list}` is a comma-separated list of platforms chosen from the
+following tags:
+
+| Tag      | Platform |
+| :----    | :------- |
+| zlinux   | 64-bit Linux on Z |
+| zos      | 64-bit zOS |
```

Comment:
Interesting. So we have no 32-bit Z targets. Anyone have an idea if this is hard to setup? I woudln't mind running all the Tril tests on 32-bit Linux on Z for example as we need coverage there.

Reply:
Interesting.  I had not noticed there was not 32 bit zOS. This is something we can likely change. The zOS build is still using the autoconf and JIT static makefiles so it should be relatively easy to switch.

Reply:
I created #3591 to track this.